doc/JSON_INFO.md

```diff
@@ -1077,7 +1088,8 @@ CBMs can be defined like this:
 "stack_size" : 8,           // (Optional) How many uses are in the above-defined volume. If omitted, is the same as 'charges'
 "fun" : 50                  // Morale effects when used
 "freezing_point": 32,       // (Optional) Temperature in F at which item freezes, default is water (32F/0C)
-"cooks_like": "meat_cooked" // (Optional) If the item is used in a recipe, replaces it with its cooks_like
+"cooks_like": "meat_cooked",// (Optional) If the item is used in a recipe, replaces it with its cooks_like
+"can_heal_anyone": false    // (Optional) Can this med be used by anyone or does it require a special mutation. See `can_heal_with` in mutation. (default: true).
```

Comment:
instead of a nwe member, this could be a flag, as it's a bool. flags are great for things that have a very common default value.